server/src/daemon/collection.rs

```diff
@@ -123,20 +125,16 @@ impl Collection {
 }
 
 impl Collection {
-    // temporarily creates a generic Collection object
-    pub fn new(label: &str, alias: &str, keyring: Arc<Keyring>) -> Self {
+    pub fn new(label: &str, alias: &str, created: u64, keyring: Arc<Keyring>) -> Self {
```

Comment:
There is no point in having DateTime, converting to seconds just convert it back here. So please use `DateTime` type here instead

Reply:
The thing is this constructor should be shared with collections you just created and collections that were already available

Reply:
done.

Reply:
> There is no point in having DateTime, converting to seconds just convert it back here. So please use `DateTime` type here instead

done.

Reply:
> The thing is this constructor should be shared with collections you just created and collections that were already available

fixed via: https://github.com/bilelmoussaoui/oo7/pull/73/commits/d759844facc8ecf68d34b19c57254b38fd96e2ec